From 0f66eb51ac4bad220e5b74fd2c4c3b2126d97f4e Mon Sep 17 00:00:00 2001
From: Minkyu Kang <mk7.kang@samsung.com>
Date: Wed, 2 Jan 2013 11:54:18 +0900
Subject: [PATCH] JQM: set the highlight as default

According to the TIZEN UX, set the highlight as default.
If don't want to fill highlight then user should be set data-highlight="fasle" explicitly.

Change-Id: I573ec253d3212cc8c56c180083f4f9ce903a369b
Signed-off-by: Minkyu Kang <mk7.kang@samsung.com>
---
 .../jquery-mobile-1.2.0/js/widgets/forms/slider.js |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/js/jquery-mobile-1.2.0/js/widgets/forms/slider.js b/libs/js/jquery-mobile-1.2.0/js/widgets/forms/slider.js
index e7f5df0..1c683df 100644
--- a/libs/js/jquery-mobile-1.2.0/js/widgets/forms/slider.js
+++ b/libs/js/jquery-mobile-1.2.0/js/widgets/forms/slider.js
@@ -65,7 +65,7 @@ $.widget( "mobile.slider", $.mobile.widget, {
 			domSlider = document.createElement( 'div' ),
 			slider = $( domSlider ),
 
-			valuebg = control.jqmData( "highlight" ) && cType !== "select" ? (function() {
+			valuebg = control.jqmData( "highlight" ) !== false && cType !== "select" ? (function() {
 				var bg = document.createElement('div');
 				bg.className = 'ui-slider-bg ' + $.mobile.activeBtnClass + ' ui-btn-corner-all';
 				return $( bg ).prependTo( slider );
-- 
1.7.9.5

